src/components/visualizations/chart/highcharts/customConfiguration.ts

```diff
@@ -162,11 +163,15 @@ function hideOverlappedLabels(chartOptions: IChartOptions) {
     return {};
 }
 
-function getShowInPercentConfiguration(chartOptions: IChartOptions) {
+function getShowInPercentConfiguration(
+    chartOptions: IChartOptions,
+    _config: IChartConfig,
+    chartConfig?: IChartConfig,
+) {
     const { yAxes = [], xAxes = [] } = chartOptions;
     const percentageFormatter = partial(formatAsPercent, 100);
 
-    const xAxis = xAxes.map((axis: any) =>
+    const xAxis = xAxes.map((axis: IHighChartAxis) =>
```

Comment:
please use [IHighchartsAxisExtend](https://github.com/gooddata/gooddata-react-components/blob/next-7.2.0/src/interfaces/HighchartsExtend.ts#L8) instead of